test/cloudformation/iam_cloudformation.yaml

```diff
@@ -104,8 +104,14 @@ Resources:
               - cloudformation:GetTemplate
               - cloudformation:GetTemplateSummary
             Resource:
-              - !Sub "arn:${AWS::Partition}:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/iam-*"
-              - !Sub "arn:${AWS::Partition}:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/eksctl-*"
+              - !Sub "arn:${AWS::Partition}:cloudformation:*:${AWS::AccountId}:stack/iam-*"
+              - !Sub "arn:${AWS::Partition}:cloudformation:*:${AWS::AccountId}:stack/eksctl-*"
+            Condition:
+              StringEquals:
+                aws:RequestedRegion: 
+                - "us-east-2"
```

Comment:
Can we make this a variable/parameter that can be set by the caller? i.e. perhaps a `CommaDelmitedList`?

Reply:
That would also make this more DRY so you wouldn't have to keep re-specifying the regions everywhere